utils/material_plots_2D.py

```diff
@@ -116,22 +143,25 @@ def main():
             title = "#eta"
         elif args.angleDef == "theta":
             title = "#theta [#circ]"
+        elif args.angleDef == "thetaRad":
+            title = "#theta [rad]"
         elif args.angleDef == "cosTheta":
             title = "cos(#theta)"
         histograms[i].GetXaxis().SetTitle(title)
         histograms[i].GetYaxis().SetTitle("#phi")
 
         histograms[i].GetZaxis().SetTitle(axis_titles[i])
 
-        if args.x0max != 0.0 and plots[i] == "x0":
-            histograms[i].SetMaximum(args.x0max)
+        if args.x0Max != 0.0 and plot == "x0":
+            histograms[i].SetMaximum(args.x0Max)
```

Comment:
```suggestion
        if args.x0max != 0.0 and plot == "x0":
            histograms[i].SetMaximum(args.x0max)
```
We still have the lowercase max, I think.